### PR TITLE
[infra] Print logs URL for base images and msan libs.

### PR DESCRIPTION
**Pull request by Dor1s**

### CODE REVIEW

File: infra/gcb/build_base_images.py [hunk 1]
```diff
@@ -64,6 +71,7 @@ def main():
       projectId='oss-fuzz-base', body=build_body).execute()
   build_id = build_info['metadata']['build']['id']
 
+  print >> sys.stderr, 'Logs:', get_logs_url(build_id)
```
Comment (inferno-chromium) [inline infra/gcb/build_base_images.py]: Why print on stderr, why not just print ? same for other one.

File: infra/gcb/build_msan_libs.py [hunk 2]
```diff
@@ -57,6 +58,7 @@ def main():
       projectId='oss-fuzz-base', body=build_body).execute()
   build_id = build_info['metadata']['build']['id']
 
+  print >> sys.stderr, 'Logs:', build_base_images.get_logs_url(build_id)
   print build_id
```
Comment (inferno-chromium) [inline infra/gcb/build_msan_libs.py]: maybe add 'Build ID:' to match your format.

File: infra/gcb/build_base_images.py (hunk 1 above)
Comment (Dor1s) [reply infra/gcb/build_base_images.py]: Just following the existing format from https://github.com/google/oss-fuzz/blob/master/infra/gcb/build.py#L345

File: infra/gcb/build_msan_libs.py (hunk 2 above)
Comment (Dor1s) [reply infra/gcb/build_msan_libs.py]: The same as above, but we can update all the scripts. Might be a good idea to move these little helpers into some common utils file.

**Pull request merged**